packages/angular_devkit/benchmark/src/default-reporter.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { logging, tags } from '@angular-devkit/core';
+import { AggregatedMetric, BenchmarkReporter, Metric } from './interfaces';
+
+export const defaultReporter = (logger: logging.Logger): BenchmarkReporter => (process, groups) => {
```

Comment:
google3 style guide always prefers
export function f() {}
over
export const f = () => {};

maybe we should do the same?

Reply:
I'm happy doing that given we actually enforce it, e.g. via linting. 

Personally I prefer `export const f = () => {};` myself, for instance. Others might too. Without the style being enforced, it's up to the reviewer whether to request it to change.

I'd be reluctant to ask request that change in a PR otherwise ready to go, and eventually a couple of these uses would go in.

Reply:
I agree we should configure the linter rather than make these decisions one-off